src/System.Data.SqlClient/src/System/Data/SqlClient/SqlDependencyUtils.cs

```diff
@@ -324,9 +324,6 @@ private void RemoveDependencyFromCommandToDependenciesHash(SqlDependency depende
         {
             lock (_instanceLock)
             {
-                List<string> notificationIdsToRemove = new List<string>();
```

Comment:
System.Data.SqlClient builds downlevel versions. This won't work downlevel.

Reply:
>  guess I need to wait for this change to reach ProjectN.

It is in ProjectN

Reply:
OK. And for SqlClient, from talking to @joperezr , it is not necessarily building too much. I will leave it as is.